tests/conftest.py

```diff
@@ -21,6 +21,25 @@
 from ploomber import Env
 import pandas as pd
 from glob import iglob
+from ploomber.cli import install
+from unittest.mock import Mock, MagicMock
+from ploomber.telemetry import telemetry
+
+
+@pytest.fixture(scope='class')
```

Comment:
Im not sure if the scope='class' is required

Reply:
How else will it apply to the functions without implicitly adding it?
(When I tried without a scope it didn't)